python/pyspark/sql/streaming/query.py

```diff
@@ -337,7 +352,6 @@ def _test() -> None:
 
     globs["spark"] = spark
     globs["sqlContext"] = SQLContext.getOrCreate(spark.sparkContext)
```

Comment:
We should probably remove `sqlContext` too (and replace the usages in the doctests to `spark`).

Reply:
Or remove it since SQLContext is deprecated.

Reply:
Good catch. Address in next commit :)